test/environment/README.md

```diff
@@ -0,0 +1,120 @@
+# Test Environment for azmi tool
+
+This PowerShell module is creating testing environment for azmi tool.
+https://github.com/SRE-PRG/azmitool
+
+## Contents
+
+- Basic Usage
+- Required permissions
+- Commands
+- Test VMs
+
+## Basic Usage
+
+Prior to using this PowerShell module you need to import it.
+
+```PowerShell
+git clone https://github.com/SRE-PRG/azmitool # skip if you already have it clonned
+
```

Comment:
missing `cd azmitool`

Reply:
Thanks for spotting it! I updated `Import-Module` command, which is enough.